Chatto/sources/Keyboard/StaticKeyboardUpdatesAnimator.swift

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
I wasn't able to find any usages of this class and protocol in this PR. Should we keep it?